### ISSUE TITLE
Direct index reads of commit timestamp values returns max timestamp in emulator

### ISSUE THREAD

**Issue by vgough** [GoogleCloudPlatform/cloud-spanner-emulator#8]
Running:
* emulator v1.1.0
* go package cloud.google.com/go/spanner v1.10.0

Seeing a case where timestamps are not being handled the same way when running against the emulator vs running against a production cluster: 

I have a table which has a timestamp as part of a composite key:

	CREATE TABLE events (
		subject STRING(MAX) NOT NULL,
		created_at TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp=true),
		data BYTES(MAX) NOT NULL,
	) PRIMARY KEY (subject, created_at DESC)

When reading data back out of the table, the `created_at` column is coming out bad, with seconds and nanoseconds epoch representations overflowing (nano=-4852116231933723624 sec=253402300799).

When using a real cloud spanner instance, the same code shows the returned date looking more normal:
`2020-10-09T22:05:22Z` => nano=1602281122432767000,  sec=1602281122

[update - the date does not print correctly, unlike initially reported, it prints out as `9999-12-31T23:59:59Z` rather than `2020-10-09...`]

**Comment by adirastogi** [comment]
Are you using a SQL query or the Read API to read the column ? Could you share a snippet of the query / code you are using to read the column ?

**Comment by vgough** [comment]
I'm using the Row API:

```go
iter := s.cli.Single().ReadUsingIndex(...)
err := iter.Do(func(row *spanner.Row) error {
   var when time.Time
   var data []byte
   if err := row.Columns(&when, &data); err != nil {
      return err
   }
   ...
}
```

**Comment by alapkarapurkar** [comment]
Thanks for the details - we have a reproduction and a fix is in the works. Updating title to reflect the underlying bug.

**Comment by alapkarapurkar** [comment]
Emulator v1.1.1 with the fix is available in gcloud 315.0.0
